controller/deployer/main.go

```diff
@@ -46,8 +47,9 @@ func main() {
 	}
 
 	pgxpool, err := pgx.NewConnPool(pgx.ConnPoolConfig{
-		ConnConfig:   pgxcfg,
-		AfterConnect: que.PrepareStatements,
+		ConnConfig:     pgxcfg,
+		AfterConnect:   que.PrepareStatements,
+		MaxConnections: 10,
```

Comment:
Is it worth putting this int in a constant and also using it when creating the worker pool?